.github/workflows/pre-release.yml

```
@@ -21,5 +21,8 @@ jobs:
 
       - name: Deploy to testpypi.org
         run: |
+          LATEST_RELEASE=${curl "https://api.github.com/repos/farridav/django-jazzmin/tags" | jq -r '.[0].name'/#v}
```

I've not managed to get this to work. I get a compile error. Maybe this is needed?
```suggestion
          LATEST_RELEASE=${curl -s "https://api.github.com/repos/farridav/django-jazzmin/tags" | jq -r '[0].name[1:]'}
```